js/runner_test.go

```diff
@@ -555,17 +555,13 @@ func TestVURunContext(t *testing.T) {
 			vu.Runtime.Set("fn", func() {
 				fnCalled = true
 
-				assert.Equal(t, vu.Runtime, common.GetRuntime(*vu.Context), "incorrect runtime in context")
-				assert.Nil(t, common.GetInitEnv(*vu.Context)) // shouldn't get this in the vu context
```

Comment:
I'm not sure if it does make sense to maintain these (or equivalent) assertions. WDYT?

Reply:
the semantics are still the same, right, just accesses differently, so why not test them, in at least a few places?